torchtext/utils.py

```diff
@@ -78,12 +78,11 @@ def _process_response(r, root, filename):
         root, filename = os.path.split(path)
 
     if not os.path.exists(root):
-        if root == ".data":
-            os.makedirs(".data")
-        else:
+        try:
+            os.makedirs(root)
+        except:
```

Comment:
To be save, `except` comes with some error message. I guess you can try `except OSError:` but please double check.

Reply:
Yes, I switched to the default `OSError`